Makefile

```diff
@@ -201,7 +201,11 @@ UI_ROOT        := $(PKG_ROOT)/ui
 SQLPARSER_ROOT := $(PKG_ROOT)/sql/parser
 
 # Ensure we have an unambiguous GOPATH.
-GOPATH := $(shell $(GO) env GOPATH)
+GOPATH := $(shell readlink -f $(shell $(GO) env GOPATH))
```

Comment:
I'm afraid this won't work on macOS.

```
$ readlink -f 
readlink: illegal option -- f
usage: readlink [-n] [file ...]
```

Reply:
Make's built-in `realpath` function might do what you need, though: https://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html

Reply:
I'm a bmake guy, so thanks for pointing me in the right direction.

Reply:
@benesch regarding CURDIR, I meant that the value of the two variables compared here:
    ifeq "$(filter $(GOPATH)%,$(CURDIR))" ""

may differ in "symlinkage" while pointing to the same (with possibly either one using the symlink'd `$HOME` value).

Reply:
Ah, I see. It appears that $(CURDIR) has internally been set to the equivalent of `$(realpath $(shell pwd))`.